Makefile

```
@@ -669,6 +672,14 @@ test-capd-junit: $(SETUP_ENVTEST) $(GOTESTSUM) ## Run unit and integration tests
 kind-cluster: ## Create a new kind cluster designed for development with Tilt
 	hack/kind-install-for-capd.sh
 
+.PHONY: kind-reset
+kind-reset: ## Destroys the kind cluster with the name $CAPI_KIND_CLUSTER_NAME.
```

```suggestion
clean-kind: ## Cleanup the kind cluster with the name $CAPI_KIND_CLUSTER_NAME.
```
Also, this target should be moved down in the file so it appears under the clean section (before clean tilt)